vms/secp256k1fx/camino_keychain.go

```diff
@@ -99,7 +99,7 @@ type TraverseOwnerFunc func(
 ) (bool, error)
 
 type TraverseAliasFunc func(
-	alias *multisig.Alias,
+	alias *multisig.AliasWithNonce,
```

Comment:
we don't need nonce here, so its better use just alias, imo

Reply:
No, actually it's used in collecting aliases to export to ShMem so AliasWithNonce needed.